recipes/configs/generate.yaml

```diff
@@ -0,0 +1,39 @@
+# Config alpaca_generate.py recipe
+#
+# To launch, run the following command from root:
+#    tune alpaca_generate \
+#    --config generate \
+#    model_checkpoint=<your_checkpoint_dir> \
+#    tokenizer_checkpoint=<your_tokenizer_dir>
+
+# Model arguments
+model:
+  _component_: torchtune.models.llama2.llama2_13b
+
+checkpointer:
+  _component_: torchtune.utils.FullModelHFCheckpointer
+  checkpoint_dir: /data/users/kartikayk/cpts/Llama-2-13b-hf/
+  checkpoint_files: [
+    pytorch_model-00001-of-00003.bin,
+    pytorch_model-00002-of-00003.bin,
+    pytorch_model-00003-of-00003.bin
+  ]
+  recipe_checkpoint: null
+  output_dir: /tmp/llama2
+  model_type: LLAMA2
+
+device: cuda
+dtype: bf16
+seed: 1234
+
+temperature: 0.8
+top_k: 300
+
+# Tokenizer arguments
+tokenizer:
+  _component_: torchtune.models.llama2.llama2_tokenizer
+  path: /tmp/llama2/tokenizer.model
+
+# Generation arguments
+prompt: "Hello, my name is"
```

Comment:
Maybe it would make sense to have this be a common default prompt. (Coming back to update this comment with examples)